app/javascript/crayons/MultiSelectAutocomplete/MultiSelectAutocomplete.jsx

```diff
@@ -659,7 +659,7 @@ export const MultiSelectAutocomplete = ({
 const optionPropType = PropTypes.shape({ name: PropTypes.string });
 
 MultiSelectAutocomplete.propTypes = {
-  labelText: PropTypes.string.isRequired,
+  labelText: PropTypes.string,
```

Comment:
This needs to be required, so that there's always an accessible label, even if it's not visible - I think you could just remove the `label` in the `tags.jsx` file and just pass in the labelText to this component

Reply:
Ohh I got it!